cocos/2d/CCParticleSystem.cpp

```diff
@@ -794,7 +786,7 @@ void ParticleSystem::update(float dt)
 
                 // translate newPos to correct position, since matrix transform isn't performed in batchnode
                 // don't update the particle with the new position information, it will interfere with the radius and tangential calculations
-                if (_batchNode)
+                if (_batchNode && PositionType::FREE != _positionType)
```

Comment:
Because matrix transform isn't performed in batchnode, so if `_batchnode` is not nullptr, here needs to add _position. But now the particle system support auto-batch, so It's not necessary to refactory ParticleBatchNode but just only to compatible with it.